about.html

```diff
@@ -0,0 +1,179 @@
+            </div>
+        </div>
+    </section>
+    <section class="history-section mt-5 mb-5" id="history">
+        <div class="container-fluid">
+            <div class="container">
+                <div class="row m-lg-5">
+                    <div class="col-md-12">
+                        <h3>Afghanistan National Team History</h3>
+                         <div class="txt-border"></div>
+                         <p>The Afghanistan national football team 
+                            is the national football team of Afghanistan and is controlled by the
+                            Afghanistan Football Federation. Founded in 1922, they played their first
+                            international game against Iran in Kabul, 1941. Afghanistan then joined
+                            FIFA in 1948 and the AFC in 1954, as one of the founding members.
+                            They play their home games at the Ghazi National Olympic Stadium in Kabul, 
+                            the capital of Afghanistan. In 2013, Afghanistan won the 2013 SAFF Championship
+                            and earned the "FIFA Fair Play Award". Afghanistan has never qualified for the AFC Asian Cup.</p>
+                    </div>
+                </div>
+                <div class="row m-lg-5 g-3">
+                    <div class="col-md-6 col-sm-12">
+                        <div class="history-bg-1">
+                            <h2 class="text-white">2003</h2>
+                            <p class="text-white">Afghanistan National Team at Ghazi Studium</p>
+                        </div>
+                    </div>
+                    <div class="col-md-6 col-sm-12">
+                        <div class="history-bg-2">
+                             <h2 class="text-white">2023</h2>
+                            <p class="text-white">Afghanistan National Team at Ghazi Studium</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="abt-footer">
+        <div class="container-fluid">
+            <div class="container">
+                <div class="row p-lg-5 ">
+                     <div class="footer d-flex justify-content-center align-items-center">
+                        <div class="col-md-3 col-sm-6">
+                            <img src="assets/images/Logo.png" alt="Afghan Team Logo" width="100px" height="120px">
+                        </div>
+                        <div class="col-md-9 col-sm-6">
+                           <h5>
+                               The Afghanistan national football team 
+                            </h5>
+                            <p>
+                            Afghanistan Football Federation. Founded in 1922, they played their first
+                           </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
**OPTIONAL** - For the mobile version, the partner section is missing on this page, please fix this. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="250" alt="jtjrj" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/73607512/34ecdba2-76db-4d68-ba16-37ab7d851f02">

Project requirement:
<img width="310" alt="wrawrhar" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/73607512/e22790e0-e83d-449f-9398-e5acd810dd8f">